File: Day09/O04Regex04.py

import re

st = "the quick brown fox jumps over the lazy dog"

res = re.search(r'fox' , st)

if res:
    print("Match found")
    print("String that got rejected :", st[:res.start()])
    print("String that Matched the regex :",res.group(0))
    print("String that is unchecked :", st[res.end():])
else:
    print("Match not found")

File: Perl_Day04/O04Regex04.pl

#! C:\perl64\bin

$st = "the quick brown fox jumps over the lazy dog";

if ($st =~ m/\bfox/)
{
	print "Match found \n";
	print "String that did not match regex 		- $` \n";
	print "String that matched the regex 		- $& \n";
	print "String that is yet to be chcecked 	- $' \n";
} else {
	print "Match not found \n";
}
